src/Microsoft.Azure.Relay/HybridConnectionConstants.cs

```diff
@@ -24,7 +24,7 @@ static class HybridConnectionConstants
         public const string SasKeyName = QueryStringKeyPrefix + "sas-key-name"; // sb-hc-sas-key-name
         public const string SasKey = QueryStringKeyPrefix + "sas-key"; // sb-hc-sas-key
 
-        public static readonly TimeSpan KeepAliveInterval = TimeSpan.FromMinutes(3.5);
+        public static readonly TimeSpan KeepAliveInterval = TimeSpan.FromMinutes(1);
```

Comment:
>FromMinutes [](start = 69, length = 11)

I'd rather make this user configurable than change everyone to use a 3x more aggressive keep-alive interval. (That would triple the amount of busy work the cloud service must do for idle websocket connections).

Reply:
Made Keep-Alive interval configurable.